migrations/6_redeploy_token_handlers.js

```diff
@@ -0,0 +1,59 @@
+// The Licensed Work is (c) 2022 Sygma
+// SPDX-License-Identifier: BUSL-1.1
+
+const parseArgs = require("minimist");
+
+const Utils = require("./utils");
+
+const BridgeContract = artifacts.require("Bridge");
+const ERC20HandlerContract = artifacts.require("ERC20Handler");
+const XC20HandlerContract = artifacts.require("XC20Handler");
+
+
+module.exports = async function (deployer, network) {
+  // check if "redeploy-token-handlers" is provided -> redeploys
+  // token handlers and registers them on bridge contract
+  const redeployTokenHandlers = parseArgs(process.argv.slice(2))["redeploy-token-handlers"];
+  if (redeployTokenHandlers) {
+    const networksConfig = Utils.getNetworksConfig();
+    // trim suffix from network name and fetch current network config
+    const currentNetworkName = network.split("-")[0];
+    const currentNetworkConfig = networksConfig[currentNetworkName];
+
+    let bridgeInstance;
+    let erc20HandlerInstance;
+    let xc20HandlerInstance;
+
+    // fetch deployed contracts addresses
+    try {
+      bridgeInstance = await BridgeContract.deployed();
+      erc20HandlerInstance = await ERC20HandlerContract.deployed();
+      xc20HandlerInstance = await XC20HandlerContract.deployed();
+    } catch(e){
+      console.error(e)
```

Comment:
Is this okay just to print out error?
I would leave the exception as it shouldn't run anything else if it fails to fetch contract instances

Reply:
If `XC20Handler` (or some other we implement in the future) isn't deployed to the current network migration an error is thrown and it stops further execution and I want it to just ignore that error and continue with other deployments/configurations. 
I was thinking about putting ERC20Handler outside try/catch since it's always deployed, and other are optional, but in the end decided to leave it as is.

Reply:
Without bridge instance this all fails as well.